### PR TITLE
Update packages

### PR DESCRIPTION
**Pull request by RyanPWalker**

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,28 +1,24 @@
...
+        className="App-link"
+        href="https://reactjs.org"
+        target="_blank"
+        rel="noopener noreferrer"
+      >
+        Learn React
+      </a>
+    </header>
+  </div>
+);
```
Comment (zmoney20) [inline src/App.js:22]: I actually just learned about this, nice clean up.

File: package.json [hunk 2]
```diff
@@ -18,50 +18,49 @@
   ],
   "scripts": {
     "bounce": "rm -rf /node_modules && yarn",
```
Comment (zmoney20) [inline package.json:19]: What does this do? Does it like clear cache or delete something?

**Review by zmoney20 — APPROVED** [review]
This is a really cool boilerplate. Honestly, there's a lot of stuff in here I don't understand but it's exciting to see! The boilerplates I've set up have been super basic but it was enough that I sort of understood what was happening!

**Review by zmoney20 — APPROVED** [review]
lookin good

File: src/App.js (hunk 1 above)
Comment (RyanPWalker) [reply src/App.js:22]: Yup best to use Functional Components instead of Classes when possible

File: package.json (hunk 2 above)
Comment (RyanPWalker) [reply package.json:19]: in a sense, yea.  I probably should just remove this cuz it's not a frequent thing I use.  It removes the installed node modules and redownloads them based on the yarn.lock file.  It's useful when you could have mismatched installed packages or tampered packages/files.

**Pull request merged**